cardano-cli/test/cardano-cli-golden/files/golden/key/non-extended-keys/non-extended-shelley.000.vkey

```diff
@@ -1,5 +1,5 @@
 {
     "type": "GenesisVerificationKey_ed25519",
-    "description": "",
+    "description": "Genesis Verification Key",
     "cborHex": "58200834b58f4bdda9522bb202af1f546db4cbbd94b068ae72c9fd96d9b55279edf0"
 }
```

Comment:
Are we missing golden tests if only two golden files have changed?

Reply:
Yes that is correct. I created a follow-up issue to add the missing tests in another PR: https://github.com/input-output-hk/cardano-cli/issues/424

Reply:
So I'm resolving this thread.